packages/eslint-plugin/src/rules/no-context-default-value/index.test.js

```diff
@@ -1,7 +1,7 @@
-const { ESLintUtils } = require('@typescript-eslint/utils');
+const { RuleTester } = require('@typescript-eslint/rule-tester');
 const rule = require('./index');
 
-const ruleTester = new ESLintUtils.RuleTester({
```

Comment:
[was removed in favor of a new package](https://github.com/typescript-eslint/typescript-eslint/pull/6816)

Reply:
fyi, maybe some internals were removed but not the `/utils` deep import api

https://typescript-eslint.io/blog/announcing-typescript-eslint-v6/#tooling-breaking-changes

<img width="746" alt="image" src="https://github.com/user-attachments/assets/b07407b5-f2b9-4aff-9702-6f932fa1bc8e">

--

im fine using the latest ofc, but in order to be consistent we need to update:
- `tools/eslint-rules` project 
- `tools/workspace-plugin/src/generators/eslint-rule/generator.ts` generator